sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialImdsLiveTests.cs

```diff
@@ -74,9 +74,7 @@ private ManagedIdentityCredential CreateManagedIdentityCredential(string clientI
         {
             options = InstrumentClientOptions(options ?? new TokenCredentialOptions());
 
-            var pipeline = CredentialPipeline.GetInstance(options);
-
-            var cred = new ManagedIdentityCredential(new ManagedIdentityClient(pipeline, clientId));
+            var cred = new ManagedIdentityCredential(clientId, options);
```

Comment:
Use the public constructor instead of the internal one, now that the public one passes through options correctly.